moped-editor/src/utils/gridTableHelpers.js

```diff
@@ -7,12 +7,20 @@
  * @param {*} value - The value in question
  * @returns {*} - The value output
  */
-export const getSearchValue = (query, column, value) => {
+
+export const parseGqlString = (value) => {
```

Comment:
i created a separate function to parse a string before it gets fed into GraphQL so that we can add any additional unrecognized characters we might find to this regex. in the longer term, should we consider mutating that queries so users can make use of double quotes to refine their searches?